apps/api/src/server/applications/s51advice/s51-advice.routes.js

```diff
@@ -337,4 +338,26 @@ router.head(
 	asyncHandler(verifyS51TitleIsUnique)
 );
 
+router.post(
+	'/:id/s51-advice/publish-queue-items',
+	/*
+        #swagger.tags = ['Applications']
+        #swagger.path = '/applications/{id}/s51-advice/ready-to-publish'
```

Comment:
swagger.path needs to match the route - this should say '/applications/{id}/s51-advice/publish-queue-items'.  Currently this duplicates ready-to-publish and means that neither can be tested correctly in the swagger api doc

Reply:
Ah good spot, I missed that. Will update now.